pac4j-http/src/main/java/org/pac4j/http/authorization/authorizer/IpRegexpAuthorizer.java

```diff
@@ -13,7 +13,7 @@
    See the License for the specific language governing permissions and
    limitations under the License.
  */
-package org.pac4j.http.authorization;
+package org.pac4j.http.authorization.authorizer;
```

Comment:
Push this new authorizer in the `authorizer` package. See: https://github.com/pac4j/pac4j/issues/337